airflow/utils/log/logging_mixin.py

```diff
@@ -134,9 +136,6 @@ def __init__(self, stream):
         if 'stdout' in stream:
             self._use_stderr = False
 
-        # StreamHandler tries to set self.stream
-        Handler.__init__(self)  # pylint: disable=non-parent-init-called
```

Comment:
Ohhh. There's a trick here. One init was intentionally omitted.

Reply:
Yeah, I found out that too, but why?